hs_core/templates/pages/genericresource.html

```diff
@@ -2674,12 +2675,14 @@ <h3 style="color: red" id="file-type-error"></h3>
                                 this.remove();
                             });
                             $(".alert-success").fadeTo(2000, 500).fadeOut(1000, function(){
+                                $(document).trigger(jQuery.Event("submit-error"));
```

Comment:
Do you need to call this submit-error event both on when the call returns success and error?